docs/docs-content/clusters/public-cloud/azure/create-azure-cluster.md

```diff
@@ -16,6 +16,15 @@ Autoscaling is not supported for Azure IaaS clusters.
 
 :::
 
+## Limitations
```

Comment:
I'm curious. Do we need this section here since we already have the same text with limitations in step 9? :)

Reply:
It's part of the template :) So, for consistency, I added them here and then also added the info under the respective step to prevent users from jumping up and down the page 👁️

Reply:
@yuliiiah can we add the autoscaler warning as a bullet to the this limitation section? That way we can reduce the number of warning boxes

Reply:
Yup, good idea! Will update now 👍🏻